.github/workflows/lango-stale-reviews.yml

```diff
@@ -0,0 +1,25 @@
+name: "lango-stale-reviews"
+on:
+  schedule:
+    - cron:  '* */1 * * *'    # At every hour.
+
+jobs:
+  update-last-activity-dates:
+    env:
+      LANGO_REVIEW_BOARD_TOKEN: ${{ github.token }}
+      ORGANIZATION: Tarantool
+      PROJECT_ID: 83
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python3
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests
```

Comment:
How long this step? I guess we can use caching here: https://github.com/actions/setup-python#caching-packages-dependencies (and I would skip updating pip itself, because I see no reason).

Reply:
Nice, thanks!